.github/workflows/check-openapi-updates.yml

```diff
@@ -12,3 +12,19 @@ jobs:
       - uses: actions/checkout@v4
       - name: Check OpenAPI updates
         run: make openapi-spec-check-updates
+  on-failure:
```

Comment:
Based on [this post](https://tomjones.dev/blog/send-github-action-workflow-failure-notifications-to-slack/).